### PR TITLE
Fixing symbol key crash.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

File: src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb [hunk 1]
```diff
@@ -1682,5 +1682,258 @@ class C
             End Using
 
         End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TargetTypePreselection1() As Task
```
Comment (Pilchie) [inline src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb]: I expect these tests will fail. The cherry-pick requires a bit of care to get just the one test you added.

**Comment by CyrusNajmabadi** [comment]
Thanks @Pilchie .  I've fixed up the bad cherry-pick.  Will this go into master automatically?

**Comment by CyrusNajmabadi** [comment]
test vsi please

**Comment by Pilchie** [comment]
:+1:

@ManishJayaswal can we get approval for `dev15-preview-4`?

**Comment by Pilchie** [comment]
@CyrusNajmabadi Yes, once merged, @dotnet/roslyn-infrastructure will take care of getting it to `master`.

File: src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb [hunk 2]
```diff
@@ -1682,5 +1682,28 @@ class C
             End Using
 
         End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TestRecursiveGenericSymbolKey() As Task
```
Comment (jasonmalinowski) [inline src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb]: Do we need this test anymore? Or still belief that there might be fancy completion logic in there too?

Comment (Pilchie) [inline src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb]: I'm not against having some redundancy in tests when one covers the end user scenario.

**Comment by jasonmalinowski** [comment]
:+1:. This makes a lot more sense!

File: src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb (hunk 2 above)
Comment (CyrusNajmabadi) [inline src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb]: I agree.  It was a brainfart that made me forget to do a symbolkey test.  But i do like an actual test that goes through the full codepath.

**Comment by CyrusNajmabadi** [comment]
retest windows_vsi_p3_open_prtest  please

**Comment by Pilchie** [comment]
Retest windows_vsi_p3_open_prtest

**Comment by Pilchie** [comment]
retest windows_vsi_p3_open_prtest please

**Pull request merged**